### DISCUSSION TITLE
add warning/error indication on device in integration panel

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#9906] [Dashboard]
Currently there is no functionality in the UI to indicate a problem on a device.
specifically noticed this because from 2021.9 z-waveJS has been more specific when indicating issues on the meter CC and multilevel sensor CC values. the logs are showing 
```
2021-08-28 13:59:42 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='81-50-2-value-590081') will be skipped: Value Value(value_id='81-50-2-value-590081') has unknown data in the following location: metadata.cc_specific.meterType. Please report this issue, it can be caused by either an upstream issue with the driver or missing support for this data in the library
```

but the UI has no pointer anything is amiss, nor even an easy way of pointing to the specific device. We have to walk through the hidden storage files to see what is happening where.

An indication in the device list of an integration would be of enormous help. (compare that the red icon in the status column on the individual devices, maybe a status column could be added to the device panel?)

see discord #devs_zwave https://discord.com/channels/330944238910963714/800356888827002880/881721267366015026

**Comment by raman325** [comment]
this particular issue has been addressed by making the log message more descriptive, but it would definitely be cool to be able to use the device registry to indicate issues with a particular device in a more visual way. It may introduce some unwanted complexity because integrations would need a way to indicate a failure as well as to "resolve" the failure, but maybe there's an elegant way to handle that.
